Tool/install.sh

```diff
@@ -7,4 +7,8 @@ INSTALL_DIR=`dirname "$INSTALL_PATH"`
 mkdir -p "$INSTALL_DIR"
 ln -sf "$TOOL_PATH" "$INSTALL_PATH"
 
```

Comment:
This would need some detailed comments explaining what exactly this does as this is some pretty advanced bash scripting.